targets/targets.json

```diff
@@ -2751,7 +2751,9 @@
         "extra_labels": ["ARM_SSG", "MPS2", "MPS2_M0"],
         "macros": ["CMSDK_CM0", "CMSIS_VECTAB_VIRTUAL", "CMSIS_VECTAB_VIRTUAL_HEADER_FILE=\"cmsis_nvic.h\""],
         "device_has": ["AACI", "ANALOGIN", "CLCD", "ETHERNET", "I2C", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "SERIAL", "SERIAL_FC", "SPI", "SPISLAVE", "TSC"],
-        "release_versions": ["2"]
+        "release_versions": ["2", "5"],
+        "copy_method": "mps2",
+        "reset_method": "reboot.txt"
```

Comment:
What does this reboot.txt file reference? I don't see it anywhere in the repo.

Reply:
@cmonr : It's a custom mod for greentea / htrun.. since CM3DS runs on an FPGA the architecture is to first burn the FPGA image on reset / power on and then flash the binary from a secondary storage. The reboot.txt handles bits of this logic. This is specific to Arm platforms since all the MPS2, MPS2+ CM3DS etc are all based on FPGA's ..